src/mode.cpp

```diff
@@ -449,6 +449,8 @@ unsigned int ModeParser::ProcessSingle(User* user, Channel* targetchannel, User*
 
 		Modes::Change& item = *i;
 		ModeHandler* mh = item.mh;
+		if (mh == NULL)
```

Comment:
Can you add a comment before this describing the contexts in which this might be null? Thanks.